content/contributing/documentation-languages.md

```diff
@@ -0,0 +1,74 @@
+
+```yaml
+-   name: en
+    lang: en
+    display_name: XRP Ledger Dev Portal
+    # These github_ fields are used by the template's "Edit on GitHub" link.
+    #  Override them with --vars to change which fork/branch to edit.
+    github_forkurl: https://github.com/XRPLF/xrpl-dev-portal
+    github_branch: master
+    strings:
+        blog: "Blog"
+        search: "Search site with Google..."
+        bc_home: "Home"
+        # ...
+```
+
+There is also a top-level `languages` listing that defines some properties for each supported language. The short code for each language should be short code according to [IETF BCP47](https://tools.ietf.org/html/bcp47). For example, "en" for English, "es" for Spanish, "ja" for Japanese, "zh-CN" for Simplified Chinese, "zh-TW" for Traditional Chinese (as used in Taiwan), and so on. The `display_name` field defines the language's name as written natively in that language. The `prefix` field defines a prefix to be used in hyperlinks to that language's version of the site. Example `languages` definition:
+
+```yaml
+languages:
+    -   code: en
+        display_name: English
+        prefix: "/"
+    -   code: ja
+        display_name: 日本語
+        prefix: "/ja/"
+```
+
+The same `dactyl-config.yml` file contains an entry for each content page in the XRP Ledger Dev Portal. If a page has been translated, there is a separate entry for each translation, linked to the "target" for that translation. If a page has not yet been translated, the English version is used across all targets.
+
+By convention, a page's the HTML filename (`html` field) should be the same across all language versions of a page. Translated Markdown source files should use the same filename as the English-language version except that the file extension should be `.{language code}.md` instead of only `.md` for English. For example, Japanese translated files end in `.ja.md`
+- **`blurb`** - a short summary of the page. This text is mostly used on category landing pages.
+
+Example of English and Japanese entries for the `server_info` method page:
+
+```yaml
+    -   md: references/http-websocket-apis/public-api-methods/server-info-methods/server_info.md
+        targets:
+            - en
+
+    -   md: references/http-websocket-apis/public-api-methods/server-info-methods/server_info.ja.md
+        targets:
+            - ja
+```
+
+Example entry for a page that isn't translated:
+
+```yaml
+    -   md: concepts/payment-system-basics/transaction-basics/source-and-destination-tags.md
+        targets:
+            - en
+            - ja
+```
+
+## Where to Start
+
+If you want to translate the XRP Ledger Dev Portal into your native language of choice, start with the [XRP Ledger Overview file](https://github.com/XRPLF/xrpl-dev-portal/blob/master/content/concepts/introduction/xrp-ledger-overview.md), which describes the core properties and functions of the XRP Ledger.
```

Comment:
When this PR is merged into the master branch, `content/concepts/introduction/xrp-ledger-overview.md` will not exist.